modules/satoshi/packer-makefile.template

```diff
@@ -0,0 +1,25 @@
+#
+# DO NOT OVERRIDE THIS FILE. AUTO-GENERATED FROM 'make satoshi/update-makefile/packer'
+#
+export HELP_FILTER ?= asdf|grafana|jsonnet|k8s|kyverno|opa|pluto|satoshi|updater
```

Comment:
if you add `packer` to this list it will show up with the makefile targets when you run `make help`